workbench-core/authentication/src/authenticationMiddleware.ts

```diff
@@ -5,12 +5,18 @@
 
 import { AuthenticatedUser, RoutesIgnored } from '@aws/workbench-core-authorization';
 import { LoggingService } from '@aws/workbench-core-logging';
-import { Request, Response, NextFunction } from 'express';
+import { Request, Response, NextFunction, CookieOptions } from 'express';
 import get from 'lodash/get';
 import has from 'lodash/has';
 import { AuthenticationService } from './authenticationService';
 import { isIdpUnavailableError } from './errors/idpUnavailableError';
 
+const defaultCookieOptions: CookieOptions = {
+  httpOnly: true,
```

Comment:
don't we want https? for default

Reply:
`httpOnly: true` prevents the cookie from being accessed by javascript code. `secure: true` only allows the cookie to be sent over https